pycon.md

```diff
@@ -228,41 +242,50 @@ This will result in installing incompatible versions.
 
 # Defensive packaging
 
-Handle everything that can happen, then deal with everything that can't.
+Handle everything that can happen,  
+deal with everything that can't.
 
 -- 
 
-## Good error messages
+## Principle #0
```

Comment:
I don't think "Principle XXX" are good title. They are not memorable. The order does not import much.